pkgs/by-name/pr/protonmail-export/package.nix

```
@@ -0,0 +1,86 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  buildGoModule,
+
+  cmake,
+  curl,
+  zip,
+  unzip,
+  go,
+
+  fmt,
+  catch2,
+  cxxopts,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "protonmail-export";
+  version = "1.0.5";
+
+  src = fetchFromGitHub {
+    owner = "ProtonMail";
+    repo = "proton-mail-export";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-rpfTI3vcZlEK1TrxRMMHFKutwC/YqAZrZCFiUsfMafc=";
+    fetchSubmodules = true;
```

```suggestion
    tag = "v${finalAttrs.version}";
    fetchSubmodules = true;
    hash = "sha256-rpfTI3vcZlEK1TrxRMMHFKutwC/YqAZrZCFiUsfMafc=";
```
Nit: move `fetchSubmodules = true;` before the `hash` as it affects the latter.